perun/postprocess/regression_analysis/methods.py

```diff
@@ -11,7 +11,7 @@
 import perun.postprocess.regression_analysis.tools as tools
 
 
-def get_supported_methods():
+def get_param_methods():
```

Comment:
Suggestion: `get_supported_parametric_methods` or `get_supported_param_methods`. I assume this was renamed to avoid clash with non-param methods. However, I'm missing the supported keyword.